src/Compilers/VisualBasic/Portable/Symbols/TypeSymbolExtensions.vb

```diff
@@ -232,12 +232,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return type.TypeKind = TypeKind.Delegate
         End Function
 
+        <Extension()>
+        Friend Function IsSameTypeIgnoringCustomModifiers(t1 As TypeSymbol, t2 As TypeSymbol) As Boolean
+            ' TODO REVIEW I should rename this method
```

Comment:
+1 For rename

---
In reply to: [81454849](https://github.com/dotnet/roslyn/pull/14229#discussion_r81454849) [](ancestors = 81454849)